tests/test_configuration_steps.py

```diff
@@ -63,10 +63,8 @@ def test_execute_configuration_step_without_service_success():
             NotificationConfigurationStep, yaml_source=CONFIG_FILE_PATH_NO_SERVICE
         )
 
-    assert (
-        "notifications_config.notifications_api_service_identifier\n  Input should be a valid string"
-        in str(excinfo.value)
-    )
+    assert "notifications_api_service_identifier" in str(excinfo.value)
+    assert "validation error" in str(excinfo.value)
```

Comment:
Let's make this a bit more specific
```suggestion
    assert "validation error" in str(excinfo.value)
    assert "Field required" in str(excinfo.value)
```